.github/workflows/scripts/mark-issues-as-stale.sh

```diff
@@ -67,11 +67,11 @@ for ISSUE in ${ISSUES}; do
     done
 
     if [[ -z "${OWNER_MENTIONS}" ]]; then
-        echo "No code owners found. Marking issue as stale without pinging code owners."
+        echo "No code owners found. Marking issue #${ISSUE} as stale without pinging code owners."
 
         gh issue comment "${ISSUE}" -b "${STALE_MESSAGE}"
     else
-        echo "Pinging code owners for issue #${ISSUE}."
+        printf "Pinging code owners for issue #${ISSUE}:\n${OWNER_MENTIONS}"
```

Comment:
This should still be an echo correct?

Reply:
I switched it to use `printf` since echo will print newlines literally and make the message hard to read.